src/components/signin/signin.css

```diff
@@ -0,0 +1,54 @@
+.logo {
+  display: block;
+  margin: auto;;
+  width: 50%;
+}
+
+.header{
+  text-align: center;
+  color: blue;
+  padding: auto;
+  margin: 50px;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif; 
+}
+
+.signupFormTitle, .signinFormTitle{
```

Comment:
The SignIn component's styles should only contain its styles and not those of the SignUp component.